.circleci/config.yml

```diff
@@ -311,6 +311,24 @@ commands:
     steps:
       - *collect_overlay_coverage
       - *upload_overlay_coverage
+  install_doc_dependencies:
+    description: "Install documentation dependencies"
+    steps:
+      - run: apk --no-cache add python3
```

Comment:
@ruffsl For these docs build steps, do you prefer that I add the low level commands to the `references` section?

Reply:
That would be nice, to help keep the `commands` section readable from a high level. Also, might want to collapse the run steps into conceptual stages or multiline run statement like the others, so that the CircleCI web UI isn't cluttered with a bunch of accordion rows for every little command.